pyquil/reference_simulator.py

```diff
@@ -166,19 +166,29 @@ def __init__(self, n_qubits: int, rs: RandomState = None):
         self.density = np.zeros((2 ** n_qubits, 2 ** n_qubits), dtype=np.complex128)
         self.density[0, 0] = complex(1.0, 0)
 
-    def sample_bitstrings(self, n_samples):
+    def sample_bitstrings(self, n_samples, tol_factor: float = 1e8):
         """
         Sample bitstrings from the distribution defined by the wavefunction.
 
         Qubit 0 is at ``out[:, 0]``.
 
         :param n_samples: The number of bitstrings to sample
+        :param tol_factor: Tolerance for setting imaginary probabilities to zero, relative to
+        machine epsilon.
```

Comment:
Tab space before `machine epsilon`

Reply:
What do you mean here? Do you want a tab space? I just deleted the space and PyCharm has reformatted it to whatever it defaults to, anyway hopefully that resolves the issue.